### PR TITLE
Bug fix pedry

### PR DESCRIPTION
**Pull request by chao2048**
View each commit for the bugs fixed.

### CODE REVIEW

**Comment by zzs-RedCocoon** [comment]
I didn't find any bug in the commits. I think we can merge it now.

File: src/main/java/ListCommand.java [hunk 1]
```diff
@@ -5,5 +5,6 @@ public class ListCommand extends Command {
     public void execute(WatchedList watchedList, ToWatchList toWatchList, Ui ui, Storage storage) {
         // list the watched list
         watchedList.list();
+        Ui.showListMessage(watchedList);
```
Comment (zzs-RedCocoon) [inline src/main/java/ListCommand.java:8]: watchedList.list() alone can list all the movie in it. Adding Ui.showListMessage() will print the list again.
`list`
`1. (#Hommes: True Story of the Thieves' World (2015)  [Documentary]`
`2. (#LANDoftheBRAVEfilm (2019)  [Thriller]`
`1. #Hommes: True Story of the Thieves' World`
`2. #LANDoftheBRAVEfilm`
We can delete one and keep one. I think deleting either of them is reasonable.

File: src/main/java/WatchedList.java [hunk 2]
```diff
@@ -13,18 +13,22 @@ public WatchedList(ArrayList<String[]> loaded) {
...
         //no movie, cannot add any review
         if (movieList.size() == 0) {
-            System.out.println("Sorry, there's no movie in watchedlist. Please add some movies first.");
+            System.out.println("Sorry, there's no movie in watched list. Please add some movies first.");
             return;
         }
 
         //get valid movie index
-        int index = Parser.parseIndex(indexString, 1, movieList.size());
+        int index = Parser.parseIndex(indexString, 1, movieList.size()+1);
```
Comment (zzs-RedCocoon) [inline src/main/java/WatchedList.java:21]: Please add space here.
`int index = Parser.parseIndex(indexString, 1, movieList.size()+1);`
->
`int index = Parser.parseIndex(indexString, 1, movieList.size() + 1);`

**Review by zzs-RedCocoon — COMMENTED** [review]
Looks good.

File: src/main/java/WatchedList.java [hunk 3]
```diff
@@ -34,7 +38,7 @@ public void setReview(String indexString) {
     }
 
     public void deleteReview(String indexString) {
-        int index = Parser.parseIndex(indexString, 1, movieList.size());
+        int index = Parser.parseIndex(indexString, 1, movieList.size()+1);
```
Comment (zzs-RedCocoon) [inline src/main/java/WatchedList.java:41]: Please add space here.

**Review by choongzhanhong — COMMENTED** [review]
LGTM

**Pull request merged**